test/special/noHighlight.js

```diff
@@ -1,6 +1,6 @@
-'use strict';
+import { document } from './document.js'
 
-describe('no highlighting', () => {
+describe('no highlighting', function() {
```

Comment:
Was this (and the other `=>` => `function` changes) necessary or is this just a style change?

Reply:
Yes, because `this` is undefined in an ES module, so the test throws when arriving at `this.blocks =`. Another way of fixing it would be to declare a variable `blocks` in the function scope and do a `s/this.blocks/blocks/` replacement.